app/policies/review_policy.rb

```diff
@@ -0,0 +1,30 @@
+class ReviewPolicy < ApplicationPolicy
+  def new?
+    create?
+  end
+
+  def create?
+    true
```

Comment:
We only need the create? method

delete everything else

Reply:
@RuiSMagalhaes done.
One quick question: what about when the booking expires? We mentioned that bookings would expires after 30 minutes, regardless.

Reply:
cc @emanuelmoniz @joaofrancosta